### PR TITLE
Added LOCATION test cases

### LINKED ISSUE #6: Detecting Location though GPS and indirect sources
One of the many types of personally identifiable information (PII) is location. Any location data that can be used to identify a person directly or indirectly is an example of a PII. For example, GPS technology can be used to directly identify a specific person, while addresses, your place of birth, and anonymous geolocation data can be used to identify and track people indirectly. Using location as a PII is important because most of the things that we do are location dependent. We live at a certain location, work at a certain location and spend our lives moving from one location to another. Location is factored into the purchases we make and determine the laws by which we have to abide. When used correctly, location data can be used for both security and authentication. However, if not properly protected, location data can fall into the hands of bad actors and affect both your virtual and physical security.

Click [here](https://storymaps.arcgis.com/stories/fea39d901cf1431184252802f33af9be) for more information on Geo-privacy.

Click [here](https://consciousdigital.org/location-data-is-personally-identifiable-information/) to see a data leak of location data from Toyota and how anonymous location data can still be used to identify individuals.

### PR DESCRIPTION
**Pull request by Twintastic**

### CODE REVIEW

**Review by claesmk — APPROVED** [review]
Looks great

**Comment by claesmk** [comment]
You are still waiting on review from you team before you can merge this. @JalensMindset

**Comment by claesmk** [comment]
@Twintastic this PR should be good to go as soon as one more person on your team reviews it.

**Comment by Twintastic** [comment]
Thanks I’ll message them then.

**Comment by Twintastic** [comment]
After merging the other pull request I am now getting a conflict.

**Comment by Twintastic** [comment]
I need help, I can’t do anything to fix it.

**Review by JalensMindset — APPROVED** [review]
Looks fantastic!

File: test_ai_aggies.py
Comment (claesmk) [inline test_ai_aggies.py]: It looks like you tried to resolve merge conflicts and introduced this line by mistake.

Comment (Twintastic) [reply test_ai_aggies.py]: ok I removed it

**Comment by Twintastic** [comment]
Thank you the conflict was resolved. @claesmk

**Pull request merged**
